Riot/Model/Room/RoomBubbleCellData.m

```diff
@@ -82,7 +88,7 @@ - (NSAttributedString*)attributedTextMessage
 {
     @synchronized(bubbleComponents)
     {
-        if (!attributedTextMessage.length && bubbleComponents.count)
+        if (self.hasAttributedTextMessage && !attributedTextMessage.length && bubbleComponents.count)
```

Comment:
I think bubbleComponents.count could not be nil if self.hasAttributedTextMessage is YES.